hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl

```diff
@@ -706,6 +706,151 @@ objects:
         creationTimestamp: null
         name: osd-oauth-templates-providers
         namespace: openshift-config
+- apiVersion: hive.openshift.io/v1alpha1
+  kind: SelectorSyncSet
+  metadata:
+    labels:
+      managed.openshift.io/gitHash: ${IMAGE_TAG}
+      managed.openshift.io/gitRepoName: ${REPO_NAME}
+      managed.openshift.io/osd: 'true'
+    name: osd-logging
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+    resourceApplyMode: sync
+    resources:
+    - apiVersion: v1
+      kind: Namespace
+      metadata:
+        annotations:
+          openshift.io/node-selector: ''
+        labels:
+          openshift.io/cluster-logging: 'true'
+          openshift.io/cluster-monitoring: 'true'
+        name: openshift-logging
+    - apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        labels:
+          csc-owner-name: installed-redhat-openshift-logging
+          csc-owner-namespace: openshift-marketplace
+        name: cluster-logging
+      spec:
+        channel: preview
+        installPlanApproval: Automatic
+        name: cluster-logging
+        source: installed-redhat-openshift-logging
+        sourceNamespace: openshift-logging
+        startingCSV: clusterlogging.4.1.4-201906271212
+    - apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        labels:
+          csc-owner-name: installed-redhat-openshift-logging
+          csc-owner-namespace: openshift-marketplace
+        name: cluster-logging
+      spec:
+        channel: preview
+        installPlanApproval: Automatic
+        name: cluster-logging
+        source: installed-redhat-openshift-logging
```

Comment:
I don't see a catalogsource in this repo with that name, is it supposed to be pre-installed in the clusters?

Reply:
Good catch, looking into this.  May be installed as part of installation via the console UI.  I will update this PR when I have an update.

Reply:
Fixed.  OperatorHub in console drops a CatalogSourceConfig, which I have included now for the 2 operators.  Also had missed the OperatorGroup and had a broken Subscription for ES.